### PR TITLE
Fix README code example for channelcoding

### PR DESCRIPTION
**Pull request by londumas**
Fix both issues : https://github.com/veeresht/CommPy/issues/90 and https://github.com/veeresht/CommPy/issues/89.

This PR fixes different issues with the example code in `channelcoding/README.md` :
- `memory` should be a numpy 1D array
- `np.array([[0o171, 0o133]])` generator matrix was producing errors, I replaced by the one in the docstring examples `np.array([[5, 7]])`
- decoded messages and demodulated messages can be padded due to the modulation type.

### CODE REVIEW

File: commpy/channelcoding/README.md [hunk 1]
```diff
@@ -106,9 +107,9 @@ noiseVar = 10**(-snrdB/10) # noise variance (power)
 
 N_c = 10 # number of trials
 
-BER_soft = np.empty((N_c,))
-BER_hard = np.empty((N_c,))
-BER_uncoded = np.empty((N_c,))
+BER_soft = -9e99*np.ones(N_c)
+BER_hard = -9e99*np.ones(N_c)
+BER_uncoded = -9e99*np.ones(N_c)
```
Comment (BastienTr) [inline commpy/channelcoding/README.md:112]: This looks good to me. Could you just explain why you removed `empty` that should work here?

Comment (londumas) [reply commpy/channelcoding/README.md:112]: `empty` works indeed, but it produces arrays filed with unknown values, I don't find it very elegant. I always prefer working with arrays filled with know values. But if you do not like it, I can remove it.

**Comment by BastienTr** [comment]
I get your point. I not keen on this initialisation as it suggest that -9e99 
is intended. I you want to know the initialisation pattern, I prefer to fill 
with `np.nan` or tick to `np.empty`. That way, it is straightforward that the 
initialisation does not matter on the following code.

File: commpy/channelcoding/README.md (hunk 1 above)
Comment (BastienTr) [reply commpy/channelcoding/README.md:112]: I get your point. I not keen on this initialisation as it suggest that -9e99
is intended. I you want to know the initialisation pattern, I prefer to fill
with `np.nan` or tick to `np.empty`. That way, it is straightforward that the
initialisation does not matter on the following code.

Comment (londumas) [reply commpy/channelcoding/README.md]: Should be fixed now

**Pull request merged**